dokan/timeout.c

```diff
@@ -64,7 +64,8 @@ BOOL DOKANAPI DokanResetTimeout(ULONG Timeout, PDOKAN_FILE_INFO FileInfo) {
 }
 
 /* Legacy KeepAlive - Remove for 2.0.0 */
-UINT WINAPI DokanKeepAlive(PDOKAN_INSTANCE DokanInstance) {
+UINT WINAPI DokanKeepAlive(PVOID instance) {
```

Comment:
Can you elaborate on the why here?

Reply:
`UINT WINAPI DokanKeepAlive(PDOKAN_INSTANCE DokanInstance)` has a body.

`UINT WINAPI DokanKeepAlive(PVOID instance)` did not have a body, only a declaration in `dokani.h`.

Some compilers seem to accept the ambiguity and map both function declarations to the same function body, but my GCC did not. The change was necessary to fix it.

Reply:
Hm, I am not sure why the prototype differs from the definition in the first place.

@Liryna I am wondering why it is different in the first place. Is there anything that speaks against making the definition in timeout.c take a `PDOKAN_INSTANCE` instead of `PVOID`?

Reply:
To make it clearer, I considered to use `UINT WINAPI DokanKeepAlive(PDOKAN_INSTANCE DokanInstance)` in `dokani.h`, but that would have broken the API as the signature of the function changes.